mintlify/uma-connect/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Introduction"
+---
+import { umaaasProductName } from '/snippets/variables.mdx'
+
+
+With {umaaasProductName}, you can send and receive real-time UMA payments without touching crypto or implementing the UMA protocol. You onboard once, integrate a single API, and we route through the right licensed providers per region to handle Lightning, FX, and instant banking off-ramps.
```

Comment:
I think we should probably mention that this product is meant for regulated financial institutions with individual users who want to send and receive P2P remittances.